datavec/datavec-python/pom.xml

```diff
@@ -47,6 +53,11 @@
             <artifactId>datavec-api</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.datavec</groupId>
+            <artifactId>datavec-local</artifactId>
+            <version>${project.version}</version>
+        </dependency>
```

Comment:
@farizrahman4u could you please check this dependency, because I can see following error from CI
```
[ERROR] [ERROR] The projects in the reactor contain a cyclic reference: Edge between 'Vertex{label='org.datavec:datavec-python:1.0.0-SNAPSHOT'}' and 'Vertex{label='org.datavec:datavec-local:1.0.0-SNAPSHOT'}' introduces to cycle in the graph org.datavec:datavec-local:1.0.0-SNAPSHOT --> org.datavec:datavec-python:1.0.0-SNAPSHOT --> org.datavec:datavec-local:1.0.0-SNAPSHOT @ 
[ERROR] The projects in the reactor contain a cyclic reference: Edge between 'Vertex{label='org.datavec:datavec-python:1.0.0-SNAPSHOT'}' and 'Vertex{label='org.datavec:datavec-local:1.0.0-SNAPSHOT'}' introduces to cycle in the graph org.datavec:datavec-local:1.0.0-SNAPSHOT --> org.datavec:datavec-python:1.0.0-SNAPSHOT --> org.datavec:datavec-local:1.0.0-SNAPSHOT -> [Help 1]
```

Reply:
what about now

Reply:
need to check it, but you can run build locally...